homeassistant/components/androidtv/media_player.py

```diff
@@ -475,6 +477,36 @@ def unique_id(self):
         """Return the device unique id."""
         return self._unique_id
 
+    async def async_get_media_image(self):
+        """Fetch current playing image."""
+        if self.state in [STATE_OFF, None]:
+            return None, None
+
+        media_data = await self.hass.async_add_executor_job(self.get_raw_media_data)
+        if media_data:
+            return media_data, "image/png"
+        return None, None
+
+    @adb_decorator()
+    def get_raw_media_data(self):
+        """Raw base64 image data."""
+        try:
+            response = self.aftv.adb_shell("screencap -p | base64")
+        except UnicodeDecodeError:
+            return None
+
+        if isinstance(response, str) and response.strip():
+            return binascii.a2b_base64(response.strip().replace("\n", ""))
+
+        return None
+
+    @property
+    def media_image_hash(self):
+        """Hash value for media image."""
+        return f"{datetime.now().timestamp()}"
+
+        return None
```

Comment:
Why do you have this `return None` here?  Did you mean for it to be something like:

```python
if some_condition:
    return f"{datetime.now().timestamp()}"
return None
```

Reply:
I originally had it checking the state ... but then removed that ... as I thought there is prob no performance hit in it then checking the image... since it has that check in it..
I removed it :)